testutil/src/test/java/org/spine3/test/VerifyShould.java

```diff
@@ -0,0 +1,1312 @@
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_contains_denied_key() {
+        final Integer key = 1;
+        Verify.denyContainsKey(key, Collections.singletonMap(key, 1));
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_is_null_in_deny_contains_key() {
+        Verify.denyContainsKey(1, null);
+    }
+
+    @Test
+    public void pass_if_map_not_contains_denied_key() {
+        Verify.denyContainsKey(1, Collections.emptyMap());
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_not_contains_entry() {
+        final Integer key = 0;
+        Verify.assertContainsKeyValue(key, 0, Collections.singletonMap(key, 1));
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_is_null_in_contains_key_value() {
+        Verify.assertContainsKeyValue(1, 1, null);
+    }
+
+    @Test
+    public void pass_if_map_contains_entry() {
+        final Integer key = 1;
+        final Integer value = 1;
+
+        Verify.assertContainsKeyValue(key, value, Collections.singletonMap(key, value));
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_collection_contains_item() {
+        final Integer item = 1;
+        Verify.assertNotContains(item, Collections.singletonList(item));
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_collection_is_null_in_not_contains() {
+        Verify.assertNotContains(1, null);
+    }
+
+    @Test
+    public void pass_if_collection_not_contains_item() {
+        Verify.assertNotContains(1, Collections.emptyList());
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_iterable_contains_item() {
+        final Integer item = 1;
+        Verify.assertNotContains(item, FluentIterable.of(item));
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_iterable_is_null_in_contains_item() {
+        Verify.assertNotContains(1, (Iterable) null);
+    }
+
+    @Test
+    public void pass_if_iterable_not_contains_item() {
+        Verify.assertNotContains(1, FluentIterable.of());
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_contains_key() {
+        final Integer key = 1;
+        Verify.assertNotContainsKey(key, Collections.singletonMap(key, 1));
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = AssertionError.class)
+    public void fail_if_map_is_null_in_not_contains_key() {
+        Verify.assertNotContainsKey(1, null);
+    }
+
+    @Test
+    public void pass_if_map_not_contains_key() {
+        Verify.assertNotContainsKey(1, Collections.emptyMap());
+    }
+
+    @Test(expected = AssertionError.class)
+    public void fail_if_former_later_latter() {
```

Comment:
`...if_former_goes_after_latter` — if that's what you wanted to say.
However I would say `if_former_goes_after_latter_in_list`.